src/components/Button/Button.stories.mdx

```diff
@@ -136,6 +150,20 @@ no text, it will automatically be configured to use the `"iconOnly"` type.
 
 <ArgsTable story="Button with Controls" />
 
+## Accessibility
+
+The Reservoir `Button` component renders an HTML `<button>` element that is accessible.
+`Button`s should be used to perform user actions or submit forms, never to link
+and go to another page. For `Button`s without text and only an icon, make sure to
+add an `aria-label` attribute. `Button`s cannot have images of text.
```

Comment:
Does the `aria-label` go on the button or on the icon in the button?

Reply:
To the button.